docs/src/main/sphinx/connector/jdbc-authentication.fragment

```diff
@@ -0,0 +1,55 @@
+Data source authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector can provide credentials for the data source connection
+in multiple ways:
+
+* inline, in the connector configuration file
+* in a separate properties file
+* in a key store file
+* as extra credentials set when connecting to Trino
```

Comment:
Capitalize the first letter in each bullet point and place a period at the end.

Reply:
Done!